CHANGELOG.md

```diff
@@ -2,7 +2,68 @@
 
 ## New Features:
 
-No changes to highlight.
+### Theme Sharing 🎨 🤝
```

Comment:
Given that we've never actually added a changelog entry for themes, I think we should have one big entry in the changelog for themes which includes both creation and sharing. But @aliabid94 can do that in his PR with the theme documentation